libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/testbase/testDsl.kt

```diff
@@ -140,7 +141,12 @@ fun TestProject.build(
         enableBuildCacheDebug,
         enableBuildScan,
         gradleVersion
-    )
+    ) + listOfNotNull(kotlinDaemonDebugPort?.let {
```

Comment:
Please pass `kotlinDaemonDebugPort` to `commonBuildSetup` so it will work for `buildAndFail()` method as well

Reply:
Good idea - I've amended this commit to do that.